server/controllers/manage/bookingsController.ts

```diff
@@ -37,12 +37,14 @@ export default class BookingsController {
 
       if (crnArr.length) {
         const person = await this.personService.findByCrn(req.user.token, crnArr[0])
+        const offences = await this.personService.getOffences(req.user.token, person.crn)
```

Comment:
should we perform some validation in the `create` method to ensure an index offence is selected?

Reply:
The validation occurs on the API side 👍

Reply:
Ah of course, I forgot that's how we do it for Manage